### ISSUE TITLE
A few errors are raised from missing guards

### ISSUE THREAD

**Issue by Flygenring** [Gillespie59/eslint-plugin-angular#402]
I have come across a couple of mishaps when running the plugin, which should be quite straightforward to fix. So far I've only seen these, which is just a matter of checking if the property exists before using it:

```
TypeError: Cannot read property 'length' of undefined
at checkDi (node_modules/eslint-plugin-angular/rules/di.js:61:26)


TypeError: Cannot read property 'match' of null
at EventEmitter.Property (node_modules/eslint-plugin-angular/rules/no-inline-template.js:33:34)
```
